### PR TITLE
Add asset types model + association with agent group

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

File: tests/runtimes/local/models/models_test.py [hunk 1]
```diff
@@ -503,3 +504,69 @@ def testAssetModels_whenMultipleAssets_shouldHaveUniqueIdsPerTable(
             session.query(models.Asset).all()[1].id
             == session.query(models.IosStore).all()[0].id
         )
+
+
+def testCreateAgentGroupWithAssetTypes_always_createsAgentGroupWithAssetTypes(
+    mocker: plugin.MockerFixture, db_engine_path: str
```
Comment (deadly-panda) [inline tests/runtimes/local/models/models_test.py:510]: Don't we need the clean_db fixture ?

File: tests/runtimes/local/models/models_test.py [hunk 2]
```diff
@@ -503,3 +504,69 @@ def testAssetModels_whenMultipleAssets_shouldHaveUniqueIdsPerTable(
...
+        web_asset_type = models.AssetType.create(type="WEB")
+        agent_group_1 = models.AgentGroup(
+            name="Group 1", description="Group 1 Description"
+        )
+        agent_group_2 = models.AgentGroup(
+            name="Group 2", description="Group 2 Description"
+        )
+        agent_group_1.asset_types.append(web_asset_type)
+        agent_group_2.asset_types.append(web_asset_type)
+        session.add_all([agent_group_1, agent_group_2])
```
Comment (deadly-panda) [inline tests/runtimes/local/models/models_test.py:551]: For this unit test to make sense, one of the agent groups should have a different asset type..

File: tests/serve_app/oxo_test.py [hunk 3]
```diff
@@ -833,6 +833,7 @@ def testQueryAllAgentGroups_always_shouldReturnAllAgentGroups(
                         description
                         createdTime
                         key
+                        assetTypes,
```
Comment (deadly-panda) [inline tests/serve_app/oxo_test.py:836]:
```suggestion
                        assetTypes
```

File: tests/runtimes/local/models/models_test.py (hunk 1 above)
Comment (elyousfi5) [reply tests/runtimes/local/models/models_test.py:510]: nope, its not needed.

**Review by 3asm — APPROVED** [review]

**Comment by 3asm** [comment]
Please resolve the conflict.

**Review by 3asm — APPROVED** [review]

**Pull request merged**